radwave/package.json

```
@@ -10,7 +10,7 @@
     "webpack-plugin-vuetify": "^2.0.1"
   },
   "scripts": {
-    "build": "vue-cli-service build",
+    "build": "./patch.sh; vue-cli-service build",
     "clean": "rimraf dist",
     "lint": "vue-cli-service lint src --no-fix",
     "serve": "vue-cli-service serve"
```

```suggestion
    "serve": ".patch.sh; vue-cli-service serve"
```
If the user runs `yarn serve` without running `yarn build`, the patch is not applied. Right now, the lines added in the patch get duplicated with each run. This doesn't break anything, and will not be a problem in production

---

Perhaps not including it here this was intentional since it gets run multiple times during development, in which case there is no problem if the user runs `yarn build` before using `yarn serve`

---

Yeah, I deliberately didn't include it so that it doesn't run every time we `yarn serve`, but forgot to mention that one needs to run it at least once. (Also just as a note, it'd need to be `./patch/sh`)

I think the real solution, which I'll change the patch script to do, is to search for the inserted `MODULE_X` line, and just do nothing if that already exists, so that this script is idempotent.